daddr_t is an unknow type for musl libc

musl-gcc output:
solaris.c:14:2: error: unknown type name ‘daddr_t’

Replace it with long.

Compiled tested only, please review.

Cc: Christophe Varoqui <christophe varoqui opensvc com>
Cc: device-mapper development <dm-devel redhat com>
Signed-off-by: Xose Vazquez Perez <xose vazquez gmail com>
---
 kpartx/solaris.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kpartx/solaris.c b/kpartx/solaris.c
index 355a6cb..01da379 100644
--- a/kpartx/solaris.c
+++ b/kpartx/solaris.c
@@ -11,7 +11,7 @@
 struct solaris_x86_slice {
 	unsigned short	s_tag;		/* ID tag of partition */
 	unsigned short	s_flag;		/* permision flags */
-	daddr_t		s_start;	/* start sector no of partition */
+	long		s_start;	/* start sector no of partition */
 	long		s_size;		/* # of blocks in partition */
 };
 
-- 
2.7.4
